### DISCUSSION TITLE
Anyone knows how to enable blocks in copilot?

### DISCUSSION THREAD

**Discussion by MorelCorp** [skills/.github#285] [Q&A]
When I get suggestions it is only one line at a time and not full blocks like presented in the examples...

**Comment by liebespaar93** [comment]
https://github.com/skills/copilot-codespaces-vscode

follow this lesson !! git skill tutorial !! 
it will be teach U!

look at stop 4 ! 

🚀 42seoul~ 🎓 kyoulee!
